### DISCUSSION TITLE
[Announcement] Homework 0 is out

### DISCUSSION THREAD

**Discussion by sujin0529** [prosyslab-classroom/cs348-information-security#470] [Announcements]
### Homework 0
Hi all, this is an announcement of homework 0.
This homework includes two programming exercises.

Here are the GitHub classroom links for each assignment:
- Homework 0-1: https://classroom.github.com/a/n7MBc767
- Homework 0-2: https://classroom.github.com/a/iL8fssTg

There is no deadline for homework 0, thus no penalty regarding the homework.
Still, we highly recommend doing the homework, if you are unfamiliar with git or GitHub.

### Github account 
We need to know your Github account for homework.
Please fill out this form: https://forms.gle/RMBc1DiSL5mgNZqq8

Thanks.

**Comment by bongjunj** [comment]
The docker image installs OCaml compiler `4.13.1`, while the CI of HW-0 assumes the compiler version to be `4.11.0`.
This is causing calling `Array.combine` function possible in my dev environment, while in the CI, the test fails with unbound value error. The function is added into `Array` module since `4.13.0`. (https://v2.ocaml.org/api/Array.html) Could you sort this out?

**Comment by KAIST-JongchanPark** [comment]
Oh, the version of CI setup is old version.

However, we will not grade your HW0-x submissions, just for your practice.
Even if CI fails, that's ok.

**Comment by tncks0121** [comment]
Will the CI from HW1 will use version `4.13.1`, or should I change the dockerfile to install `4.11.0`?

**Comment by m-spitfire** [comment]
In my case, the CI for HW0 already uses `4.13.1`:
![image](https://user-images.githubusercontent.com/66242799/222065587-9e1bc506-3ceb-4e7f-b7aa-bebeea3594c0.png)

I am guessing it's already updated for other HWs too.

**Comment by tncks0121** [comment]
Thanks for the clarification :) I didn't spot the change since I created the repository yesterday. I should change the version by myself
